tablo/models.py

```diff
@@ -771,10 +801,17 @@ def update_feature(self, feature):
                 )
                 c.execute(set_geom_command, [primary_key])
 
+        # save out large images
+        for key, value in images_large.items():
+            image_path = key.replace("NO_PRIMARY_KEY", str(primary_key))
+            FeatureServiceLayer.save_image(value, image_path, LARGE_IMAGE_NAME)
+
         return primary_key
 
     def delete_feature(self, primary_key):
 
+        from django.core.files.storage import default_storage as image_storage
```

Comment:
Probably not. Is there any advantage to doing it this way? I saw several code examples doing this.